--- a/midp/build/common/makefiles/SubsystemDefs.gmk
+++ b/midp/build/common/makefiles/SubsystemDefs.gmk
@@ -330,6 +330,16 @@
 JTWI_NATIVE_FILES	+= $(SUBSYSTEM_JSR_256_NATIVE_FILES)
 endif
 
+ifeq ($(USE_JSR_184), true)
+# JSR 184 Mobile 3D Graphics
+#
+$(MIDP_CLASSES_ZIP):: $(SUBSYSTEM_JSR_184_JAVA_FILES)
+	$(appendjavafiles)
+
+JTWI_I3TEST_JAVA_FILES	+= $(SUBSYSTEM_JSR_184_I3TEST_JAVA_FILES)
+JTWI_NATIVE_FILES	+= $(SUBSYSTEM_JSR_184_NATIVE_FILES)
+endif
+
 ifeq ($(USE_JSR_172), true)
 # JSR 172 WSS
 #
